models/nxos/lacp_interfaces/nxos_lacp_interfaces.yaml

```diff
@@ -0,0 +1,100 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+NETWORK_OS: nxos
+RESOURCE: lacp_interfaces
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: nxos_lacp_interfaces
+  version_added: 2.9
+  short_description: Manage Link Aggregation Control Protocol (LACP) attributes of interfaces on Cisco NX-OS devices.
+  description: This module manages Link Aggregation Control Protocol (LACP) attributes of NX-OS Interfaces.
+  author: Trishna Guha (@trishnaguha)
+  notes:
+    - Tested against <network_os> 7.3.(0)D1(1) on VIRL
+  options:
+    config:
+      description: LACP configurations for interface.
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+            - Name of the interface.
+          type: str
+        port_priority:
+          description:
+            - LACP port priority for the interface. Range 1-65535.
+              Applicable only for Ethernet.
+          type: int
+        rate:
+          description:
+            - Rate at which PDUs are sent by LACP. Applicable only for Ethernet.
+              At fast rate LACP is transmitted once every 1 second.
+              At normal rate LACP is transmitted every 30 seconds after the link is bundled.
+          type: str
+          choices: ['fast', 'normal']
+        links:
+          description:
+            - This dict contains configurable options related to max and min port-channel links.
+              Applicable only for Port-channel.
+          type: dict
+          suboptions:
+            max:
+              description:
+                - Port-channel max bundle.
+              type: int
+            min:
+              description:
+                - Port-channel min links.
+              type: int
+        mode:
+          description:
+            - LACP mode. Applicable only for Port-channel.
+          type: str
+          choices: ['delay']
+        suspend_individual:
+          description:
+            - port-channel lacp graceful convergence.Disable this only with lacp ports
+              connected to Non-Nexus peer. Disabling this with Nexus peer can lead
+              to port suspension. Applicable only for Port-channel.
+          type: bool
+        convergence:
+          description:
+            - This dict contains configurable options related to max and min port-channel links.
+              Applicable only for Port-channel.
```

Comment:
is this description correct? seems a repeat of the links description

Reply:
updated